lib/controller/EetplanController.class.php

```diff
@@ -177,19 +184,22 @@ public function json() {
     public function nieuw() {
         $form = new NieuwEetplanForm();
 
-        if ($form->validate()) {
-            $avond = filter_input(INPUT_POST, 'avond', FILTER_SANITIZE_STRING);
+        if (!$form->validate()) {
+            $this->view = $form;
+        } elseif ($this->model->count("avond = ?", array($form->getValues()['avond'])) > 0) {

```

Comment:
Wil je toch liever de ->exist() functie terug? Die is sneller en was hier voor bedoeld.